doc/SDK_NOTIFICATIONS.md

```diff
@@ -0,0 +1,77 @@
+# Breez SDK Notifications
+Applications have their own way of delivering notifications to end users such as: mobile push, web push, emails, text messages etc...
+This is usually implemented using some backend service that has a way to communicate with the end user. The SDK goal is to provide the trigger including the type and notification data to that service. A simple and standard option to implement that trigger is using webhooks.
+
+## The use cases:
+
+### Offline payment
+User is offline and would like to be notified to accept a received payment.
+- The client associate its node id with the webhook by calling a dedicated API endpoint of the LSP.
+- The sender pays, htlcs land at the LSP which detects the last hop is not connected as a peer and send a POST request to the associated webhook with the payload that includes the notification type and data.
+- The web hook notifies the delivery service that builds the specific notification format and payload and send it to the device. In that case the push message can be silent, waking up the app to run the signer and process the payment.
```

Comment:
Brainstorm: There's a period here where the LSP will hold the htlc and will wait for the client to 'respond' to the webhook by waking up and connecting. Timeout would be the simplest option here.